crates/biome_js_formatter/tests/specs/prettier/js/multiparser-css/issue-5697.js.snap

```diff
@@ -0,0 +1,71 @@
+---
+source: crates/biome_formatter_test/src/snapshot_builder.rs
+info: js/multiparser-css/issue-5697.js
+---
+# Input
+
+```js
+const StyledH1 = styled.div`
+  font-size: 2.5em;
+  font-weight: ${(props) => (props.strong ? 500 : 100)};
+  font-family: ${constants.text.displayFont.fontFamily};
+  letter-spacing: ${(props) => (props.light ? '0.04em' : 0)};
+  color: ${(props) => props.textColor};
+  ${(props) =>
+    props.center
+      ? ` display: flex;
+                align-items: center;
+                justify-content: center;
+                text-align: center;`
+      : ''}
+  @media (max-width: ${(props) => (props.noBreakPoint ? '0' : constants.layout.breakpoint.break1)}px) {
+    font-size: 2em;
+  }
+`;
+
+```
+
+
+# Prettier differences
+
+```diff
+--- Prettier
++++ Biome
+@@ -11,8 +11,7 @@
+                 justify-content: center;
+                 text-align: center;`
+       : ""}
+-  @media (max-width: ${(props) =>
+-    props.noBreakPoint ? "0" : constants.layout.breakpoint.break1}px) {
++  @media (max-width: ${(props) => props.noBreakPoint ? "0" : constants.layout.breakpoint.break1}px) {
```

Comment:
This mismatch occurs because `Prettier` treats the template string as `CSS` since the template string is prefixed with `styled.div`  but Biome treats it as a normal template string.